src/Utils/formatters.js

```diff
@@ -0,0 +1,2 @@
+export const showDate = date => `${date.getDate()}/${date.getMonth()}/${date.getFullYear()}`;
+export const createNewsObject = (name, image, createdAt) => ({name, image, createdAt});
```

Comment:
createNewsObject should be on news service because its bussiness logic.

Reply:
Applied change